module/VuFind/src/VuFind/GeoFeatures/BasemapConfig.php

```diff
@@ -90,76 +71,61 @@ public function __construct(\VuFind\Config\PluginManager $configLoader)
      */
     public function getBasemap($origin)
     {
+        $validFields = ['basemap_url', 'basemap_attribution'];
```

Comment:
If $validFields is always going to be the same thing for all getOptions calls made by this class, perhaps it would be better to set it as a property... then you could use $this->validFields in all of the getOptions calls, and avoid the need to pass the value down to getMapSelectionBasemap and getMapTabBasemap. Might improve readability a little bit and also potentially increase extensibility (though I don't think it's incredibly likely that anyone would care about overriding it).